Library/Homebrew/downloadable.rb

```diff
@@ -5,19 +5,19 @@
 require "checksum"
 require "download_strategy"
 
-class Downloadable
+module Downloadable
```

Comment:
Is there a reason for this change? I ask as it might make the surrounding changes easier to understand.

Reply:
There are some classes that should be `Downloadable` which are already subclasses of other classes.

Reply:
I'm pretty sure there was at least one case, maybe it's gone now after rebasing, or maybe I gave up because it was too much for this PR. :shrug: